universe/ValueRefs.cpp

```diff
@@ -1613,6 +1674,26 @@ int ComplexVariable<int>::Eval(const ScriptingContext& context) const
         }
         return 0;
     }
+    else if (variable_name == "Named") {
+        if (!m_string_ref1)
+            return 0;
+        std::string rule_name = m_string_ref1->Eval();
```

Comment:
`rule_name` is misleading / confusing

Reply:
Removed "Named" complex value ref completely